vscode-sbcl-clpm.md

```diff
@@ -0,0 +1,472 @@
+reach me at hello at smashedtoatoms dot com if you run into trouble.
+
+![](assets/commonlisp-vscode-alive.png)
+
+### Prerequisites And Explanations
+
+- [VsCode](https://code.visualstudio.com) with [command
+  line](https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line)
+  installed
+- [Alive](https://marketplace.visualstudio.com/items?itemName=rheller.alive)
+  Common Lisp plugin for VsCode
+- [SBCL](http://www.sbcl.org)
+- [CLPM](https://www.clpm.dev)
+- [rlwrap](https://github.com/hanslub42/rlwrap)
+
+#### Why Use SBCL?
+
+You can honestly use whatever you want. The nice thing about Common Lisp
+being a standard is all of the implementations will all run your code
+pretty much the same. They all have slightly different goals, and each
+is a little different where the standard is not defined, but you can
+deal with it when you encounter it. I use SBCL because it is updated
+regularly, it is fast, and it is easy. If you're new to Common Lisp,
+just start with it. You can change it out later when you have your arms
+around the whole Common Lisp thing.
+
+#### Why use CLPM?
+
+CLPM has [very well-defined goals](https://www.clpm.dev/#project-goals).
+It doesn't force the consumers of your code to use it. It's written in
+Common Lisp, so it's easy to hack on if you feel so inclined.
+
+CLPM gives you deterministic builds via version pinning in projects with
+no other dependencies. It allows different projects on your computer to
+pin different versions of the same library. It has a lockfile! It's
+common in the Common Lisp world to share dependencies that update once a
+month across projects because everything is stable. I prefer more
+granular control.  [Qlot](https://github.com/fukamachi/qlot) can do this
+for you as well, though it is tricky to use without
+[Roswell](https://github.com/roswell/roswell).
+
+#### Why use rlwrap?
+
+rlwrap makes using command line utilities and repls that don't support
+history less irritating.  SBCL is one of those utilities.
+
+#### How to install SBCL, CLPM, and rlwrap
+
+- **SBCL:** You can install SBCL via the official installer, the package
+  manager provided by your OS, [homebrew](https://brew.sh/),
+  [macports](https://www.macports.org/),
+  [asdf-vm](https://github.com/smashedtoatoms/asdf-sbcl), etc.
+- **CLPM** You can install CLPM by [following the
+  instructions](https://www.clpm.dev/#installing).  If you want to skip
+  a few steps, you can download the binary from that site, drop it in
+  your path, and run
+  ```sh
+  curl -L https://smashedtoatoms.com/rando/configure-clpm-to-work-with-sbcl.sh | bash
+  ```
+  To uninstall, delete it from your path and run
+  ```sh
+  curl -L https://smashedtoatoms.com/rando/configure-clpm-to-work-with-sbcl.sh | bash -s -- cleanup
+  ```
+  If running shell scripts off the internet gives you the creeps, just
+  [follow the instructions](https://www.clpm.dev/#installing)
+- **rlwrap** You can install rlwrap using your OS package manager
```

Comment:
This section is a duplicate of the getting started page. We should refer readers there, and expand that page if needed.

Reply:
Common style corrections throughout: REPL. Emacs. Internet.

Reply:
Man, those were all over the place.  I think I got them, but if you see, them, feel free to call them out and I will fix them.  Are those in the style guide?  If they are, I totally missed them.  Sorry about that.